ui/package.json

```diff
@@ -117,6 +117,7 @@
     "date-fns-tz": "^1.2.2",
     "deepmerge": "^4.0.0",
     "doctoc": "^2.2.0",
+    "dompurify": "^3.0.2",
```

Comment:
Should this be moved from `devDependencies` to `dependencies` since it is not related to development?

Reply:
I thought that Ember prefers most dependencies in the `devDependencies` 🤔 Happy to move it but for my own learning would love to know what the actual rule is for which goes where

Reply:
Hm 🤔 that doesn't match the items we have currently in the `devDependencies` list. I just found [this discussion](https://stackoverflow.com/questions/42159153/why-does-ember-installs-everything-as-devdependencies-instead-of-normal-dependen) and it looks like they're all in dev because you don't need the imports after build

Reply:
Ok so then we probably shouldn't have anything under dependencies then. Anyway it's not important in the scope of this work but I will bring it up again at a sync. Thanks.